src/core/evaluator.js

```diff
@@ -422,6 +442,10 @@ class PartialEvaluator {
       filename = standardFontNameToFileName[name];
     let data;
 
+    if (!filename) {
+      return null;
+    }
+
```

Comment:
What is this code for, since that's not immediately obvious to me?
If this is indeed necessary, it requires a very good comment explaining things.